page-weather/style.css

```diff
@@ -0,0 +1,430 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```

Comment:
Something wrong with the main bg for some mobile views:
![image](https://user-images.githubusercontent.com/11545672/113513244-68fa2880-9571-11eb-97d9-55d7f6b044c2.png)

Reply:
![image](https://user-images.githubusercontent.com/11545672/113513276-9cd54e00-9571-11eb-8dc6-934c951d4ba1.png)

Reply:
![image](https://user-images.githubusercontent.com/11545672/113513491-b2974300-9572-11eb-8a05-e0498ae580e0.png)
Pay attention as this elements look like on layout in the task.

Reply:
Something going wrong with the weather icon:
![image](https://user-images.githubusercontent.com/11545672/113513650-6b5d8200-9573-11eb-8b5f-01ef534d8e60.png)